lesson-14/src/assets/style/style.css

```
@@ -0,0 +1,34 @@
+body {
+  position: relative;
+  font-family: Inter, Arial, sans-serif;
+  font-size: 16px;
+  color: white;
+  background-color: black;
```

```suggestion
  font-family: Inter, Arial, sans-serif;
  font-size: 16px;
  line-height: 1.6;
  color: white;
  background-color: black;
```

- position лишний
- не забывай указывать line-height по-умолчанию